src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractCreatePdpImportServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package edu.cornell.kfs.concur.batch.service.impl;
+
+import edu.cornell.kfs.concur.batch.service.ConcurEventNotificationProcessingService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpImportService;
+import edu.cornell.kfs.concur.service.ConcurRequestExtractFileService;
+
+public class ConcurRequestExtractCreatePdpImportServiceImpl implements ConcurRequestExtractCreatePdpImportService {
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpImportServiceImpl.class);
+	protected ConcurRequestExtractFileService concurRequestExtractFileService;
+
+	@Override
+    public void convertConcurRequestExtractsToPdpImports() {
```

Comment:
good point. I agree that we should call it createPdpFeedsFromRequestExtracts. This should however call another method that processed one input request extract and creates one pdp feed.